### PR TITLE
Enable deployment on gh pages

### PR DESCRIPTION
**Pull request by rldefa**
To deploy, use the command `yarn run deploy`. Remember to `yarn` first to ensure the `gh-pages` dependency is installed.

### CODE REVIEW

**Comment by rldefa** [comment]
[deploy website onto github pages](https://trello.com/c/WZTuHmND/28-deploy-website-onto-github-pages)

**Review by hui444 — APPROVED** [review]
check dependencies, otherwise lgtm

File: yarn.lock [hunk 1]
```diff
@@ -3353,6 +3365,7 @@ css-select@^2.0.0:
     boolbase "^1.0.0"
     css-what "^3.2.1"
     domutils "^1.7.0"
+    nth-check "^1.0.2"
```
[the remark this answers is not in the document]
Comment (rldefa) [reply yarn.lock:3368]: tried removing, it comes back everytime `yarn` is executed

[the remark this answers is not in the document]
Comment (rldefa) [reply yarn.lock:3368]: tried changing it to a different version too

**Pull request merged**